Library/Formula/metashell.rb

```diff
@@ -10,20 +11,28 @@ class Metashell < Formula
     sha1 "76874d93ced1815ccfcbddb6da75e8c56aff44e4" => :mountain_lion
   end
 
-  depends_on "cmake" => :build
-
   needs :cxx11
 
+  # This patch is required because Mountain Lion uses an old compiler which breaks
+  # compiling some Templight code. The patch comments out unused parts of Templight,
+  # so patched version is functionally equivalent. This error should be fixed in
+  # the next release of Metashell.
+  patch :DATA if MacOS.version == :mountain_lion
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!

Also, Mountain Lion is using the latest GCC which isn't an old compiler :wink: